### PR TITLE
excape keyword for parameter name

### PR DESCRIPTION
**Pull request by lohanidamodar**

### CODE REVIEW

File: templates/kotlin/src/main/kotlin/io/appwrite/services/ServiceTemplate.kt.twig [hunk 1]
```diff
@@ -32,19 +32,19 @@ class {{ service.name | caseUcfirst }}(private val client: Client) : BaseService
...
+     * @param {{ parameter.name | caseCamel | escapeKeyword }}
 {% endfor %}
      * {% if method.type != "webAuth" %}@return [Response]{% endif %}
      
      */
     @JvmOverloads
     @Throws({{ spec.title | caseUcfirst }}Exception::class)
     suspend fun {{ method.name | caseCamel }}({% if method.type == "webAuth" %}{{ '\n\t\t' }}activity: ComponentActivity{% if method.parameters.all | length > 0 %}, {% endif %}{% endif %}{{ _self.method_parameters(method.parameters) }}{% if method.parameters.all|length > 0 %}{{ '\n\t' }}{% endif %}){% if method.type != "webAuth" %}: Response{% endif %} {
-        val path = "{{ method.path }}"{% for parameter in method.parameters.path %}.replace("{{ '{' ~ parameter.name | caseCamel ~ '}' }}", {{ parameter.name | caseCamel }}){% endfor %}
+        val path = "{{ method.path }}"{% for parameter in method.parameters.path %}.replace("{{ '{' ~ parameter.name | caseCamel | escapeKeyword ~ '}' }}", {{ parameter.name | caseCamel | escapeKeyword }}){% endfor %}
```
Comment (abnegate) [inline templates/kotlin/src/main/kotlin/io/appwrite/services/ServiceTemplate.kt.twig:43]:
```suggestion
        val path = "{{ method.path }}"{% for parameter in method.parameters.path %}.replace("{{ '{' ~ parameter.name | caseCamel ~ '}' }}", {{ parameter.name | caseCamel | escapeKeyword }}){% endfor %}
```

File: templates/php/src/Services/Service.php.twig [hunk 2]
```diff
@@ -17,47 +17,47 @@ class {{ service.name | caseUcfirst }} extends Service
...
-        if (empty(${{ parameter.name | caseCamel }})) {
-            throw new {{spec.title | caseUcfirst}}Exception('Missing required parameter: "{{ parameter.name | caseCamel }}"');
+        if (empty(${{ parameter.name | caseCamel | escapeKeyword }})) {
+            throw new {{spec.title | caseUcfirst}}Exception('Missing required parameter: "{{ parameter.name | caseCamel | escapeKeyword }}"');
         }
 
 {% endif %}
 {% endfor %}
-        $path   = str_replace([{% for parameter in method.parameters.path %}'{{ '{' }}{{ parameter.name | caseCamel }}{{ '}' }}'{% if not loop.last %}, {% endif %}{% endfor %}], [{% for parameter in method.parameters.path %}${{ parameter.name | caseCamel }}{% if not loop.last %}, {% endif %}{% endfor %}], '{{ method.path }}');
+        $path   = str_replace([{% for parameter in method.parameters.path %}'{{ '{' }}{{ parameter.name | caseCamel | escapeKeyword }}{{ '}' }}'{% if not loop.last %}, {% endif %}{% endfor %}], [{% for parameter in method.parameters.path %}${{ parameter.name | caseCamel | escapeKeyword }}{% if not loop.last %}, {% endif %}{% endfor %}], '{{ method.path }}');
```
Comment (abnegate) [inline templates/php/src/Services/Service.php.twig:37]:
```suggestion
        $path   = str_replace([{% for parameter in method.parameters.path %}'{{ '{' }}{{ parameter.name | caseCamel }}{{ '}' }}'{% if not loop.last %}, {% endif %}{% endfor %}], [{% for parameter in method.parameters.path %}${{ parameter.name | caseCamel | escapeKeyword }}{% if not loop.last %}, {% endif %}{% endfor %}], '{{ method.path }}');
```

**Review by abnegate — COMMENTED** [review]
I don't think the path parameter string names should be escaped for Kotlin and PHP, otherwise looks good

**Review by abnegate — APPROVED** [review]
Looks good :+1:

**Review by TorstenDittmann — APPROVED** [review]

**Pull request merged**